src/main/java/net/runelite/rsb/methods/Bank.java

```diff
@@ -240,7 +240,7 @@ public boolean depositAllEquipped() {
 	 * @param items The array of items.
 	 * @return The sum of the stacks of the items.
 	 */
-	public int getCount(final Integer... items) {
+	public int getCount(final int... items) {
```

Comment:
This has to be decided once for all wherever we use Integer or int.

I personally prefer int, but since @rscorpx needs the extra flexibility of Integer lets make it integer then.

Reply:
unbiased question: 
not saying RL is *the way to salvation* but after glancing at their online api-docs and current codebase i notice they do seemingly favor the primatives for arg types over the wrappers.    

i'm all in favor for adding convenience, however, will we be faced higher instances of converting our parameter types? in my case (the reason for this push) was dealing with spread operator on int and Integer.  one does not play well with the other.

seems the RL base is leaning heavy on the primative type esp for the stuff we leverage ;/

any thoughts / feedback?